src/utils/string.js

```diff
@@ -0,0 +1,5 @@
+import {deburr} from 'lodash-es'
+
+export function normalizeString(string) {
+  return deburr(string?.toLowerCase())
+}
```

Comment:
Pour éviter d'exécuter `deburr` sur du `null` ou du `undefined`.

```suggestion
export function normalizeString(string) {
  if (!string) {
    return string
  }

  return deburr(string.toLowerCase())
}
```

Reply:
Oui, j'avais fait ça au début mais je trouve ça un peu bizarre.